src/bundle/Resources/config/routing.yml

```diff
@@ -4,8 +4,17 @@ ezplatform.dashboard:
         _controller: 'FrameworkBundle:Template:template'
         template: 'EzPlatformAdminUiBundle:dashboard:dashboard.html.twig'
 
+_ezpublishLocation:
+    path: /content/location/{locationId}/{language}/{viewType}/{layout}
```

Comment:
I would be more specific with parameter names - `language` is too generic, especially because Language uses either unique `languageCode` or `languageId`. You should call it `languageCode` in this case.